explanations/README.md

```diff
@@ -1,11 +1,10 @@
-# Explanations of internal `lcm` concepts
+# Explanations of internal `pylcm` concepts
```

Comment:
Do we always want to lowercase? I don't care too much, maybe ignore my comment above.

Reply:
I propose to follow PyTorch's approach:

- When talking about the library we write camel-case without monospace: PyLCM (PyTorch)
- When talking about the module that is imported we write lowercase with monospace: `lcm` (`torch`)